modules/xml/xml.h

```diff
@@ -30,6 +30,7 @@ typedef struct
 {
   LogParser super;
   gchar *prefix;
+  gchar *text_separator;
```

Comment:
Just a question: 
We already use g_string_append_len when appending values with separator.
It's just we have to call strlen on the `char *separator`.
So, we wouldn't save the "scanning for terminating zero", because that's already save, but
we save the time `strlen()` time, right?

Reply:
Yes, it is `strlen` who does the scanning.